izumi-reflect/izumi-reflect/src/main/scala_3/izumi/reflect/dottyreflection/Inspector.scala

```diff
@@ -122,30 +122,26 @@ abstract class Inspector(protected val shift: Int) extends InspectorBase {
   }
 
   private def prefixOf(symbol: Symbol): Option[AppliedReference] = {
-    if (symbol.maybeOwner.isNoSymbol) {
+    val mo = symbol.maybeOwner
+    if (!mo.exists || mo.isNoSymbol || mo.isPackageDef || mo.isValDef) {
```

Comment:
@lvitaly `mo.isValDef` is a correct case I think, izumi-reflect does count value prefixes, e.g. for `trait A { trait B }; val a = new A{}; Tag[a.B]` the prefix is `a`

Reply:
@neko-kai I believe you are right. I checked `inspectSymbol` function and didn't find case for `ValDef`. That's why I excluded it by `mo.isValDef`. If I get you right then `case v: ValDef => NameReference(v.name)` would be enough. Or I'm wrong? Please advise me on how to proceed.

Reply:
@lvitaly Yep, that would work correctly.